spec/generators/model_generator_spec.rb

```diff
@@ -9,6 +9,15 @@
   let(:use_fx_args) { USE_FX ? [] : ["--fx"] }
   let(:fx_args) { USE_FX ? ["--no-fx"] : [] }
 
+  let(:table_name_prefix) { TABLE_NAME_PREFIX }
+  let(:table_name_suffix) { TABLE_NAME_SUFFIX }
+
+  let(:full_table_name) {
```

Comment:
I think, we can define a plain Ruby method here:

```ruby
def full_table_name(table_name)
  # ...
end
```

Reply:
Perfect @palkan I will look into the stuff that you mentioned! thanks for the time of looking into it!